repo/packages/R/riak-ts/0/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "packagingVersion": "2.0",
+  "name": "riak-ts",
+  "version": "1.3.0-ts-1.3.1",
+  "framework": true,
+  "tags": ["mesosphere", "framework", "database", "riak", "riak-ts", "time-series"],
```

Comment:
please remove the tag `mesosphere` since this package is not developed by mesosphere.